packages/datagateway-dataview/src/views/card/datasetCardView.component.test.tsx

```diff
@@ -167,10 +166,9 @@ describe('Dataset - Card View', () => {
 
   it('updates sort query params on sort', () => {
     const wrapper = createWrapper();
-
-    const button = wrapper.find(ListItemText).first();
+    const button = wrapper.find('[aria-label="Sort by DATASETS.NAME"]').first();
     expect(button.text()).toEqual('datasets.name');
-    button.find('div').simulate('click');
+    button.find('div').last().simulate('click');
```

Comment:
This method was no longer working for some reason, hence the change in this file and others. It's a temporary fix, at least until things are refactored with `testing-library`.